<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#FFD700</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>93</x>
      <y>87</y>
      <width>291</width>
      <height>231</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#FFC700;border:2px solid white;border-radius:5px;
padding:5;</string>
    </property>
    <property name="title">
     <string>Create Process</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font-family:&quot;Times New Roman&quot;;
font-size:16px;
</string>
         </property>
         <property name="text">
          <string>Burst Time</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="BTime">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white;</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font-family:&quot;Times New Roman&quot;;
font-size:16px;
</string>
         </property>
         <property name="text">
          <string>Arrival Time</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="ATime">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#00FF7F;border:2px solid #00FF7F;border-radius:10px;;
font-size:16px;
font-family: &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Add Process</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">font-family:&quot;Times New Roman&quot;;
font-size:16px;
</string>
         </property>
         <property name="text">
          <string>Priority</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="priority_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white;</string>
         </property>
         <property name="minimum">
          <number>-20</number>
         </property>
         <property name="maximum">
          <number>19</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>300</width>
      <height>511</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>400</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#FFFFFF;border:1px solid #FFFFFF;border-radius:30;</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>298</width>
       <height>509</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="process_layout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>490</y>
      <width>261</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#00FF7F;border:2px solid #00FF7F;border-radius:10px;;
font-size:16px;
font-family: &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Exectute</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>181</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#FFC700;border:2px solid #FFFFFF;border-radius:10px;</string>
    </property>
    <property name="title">
     <string>Choose Algorithm</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="FCFS">
       <property name="styleSheet">
        <string notr="true">font-size:12px;
font-family: &quot;Segoe UI&quot;;border:none;</string>
       </property>
       <property name="text">
        <string>First Come First Serve</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="SJF">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size:12px;
font-family: &quot;Segoe UI&quot;;border:none;</string>
       </property>
       <property name="text">
        <string>Shortest Job First</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Round_robin">
       <property name="styleSheet">
        <string notr="true">font-size:12px;
font-family: &quot;Segoe UI&quot;;border:none;</string>
       </property>
       <property name="text">
        <string>Round Robin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Round_robin_2">
       <property name="styleSheet">
        <string notr="true">font-size:12px;
font-family: &quot;Segoe UI&quot;;border:none;</string>
       </property>
       <property name="text">
        <string>Priority Scheduling</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="quantum_input"/>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>570</y>
      <width>261</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#00FF7F;border:2px solid #00FF7F;border-radius:10px;;
font-size:16px;
font-family: &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Clear Process Queue</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
